client/state/inline-help/selectors.js

```diff
@@ -49,9 +45,21 @@ export function isRequestingInlineHelpSearchResultsForQuery( state, searchQuery
  * @return {?Array}         List of results for a given search query
  */
 export function getInlineHelpSearchResultsForQuery( state, searchQuery ) {
-	const searchResults = state.inlineHelpSearchResults.search.items[ searchQuery ];
+	const searchResults = get( state, 'inlineHelpSearchResults.search.items.' + searchQuery );
 	if ( ! searchResults ) {
 		return null;
 	}
 	return searchResults;
 }
+
+/**
+ * Returns the link / href of the selected search result item
+ * @param  {Object}  state  Global state tree
+ * @return {String}         The href of the selected link target
+ */
+export function getInlineHelpCurrentlySelectedLink( state ) {
```

Comment:
If we rename `getSelectedResult` to `getSelectedResultIndex`, this could make sense as `getSelectedResult`?